### PR TITLE
fix: add `pregenerate+d` to en-shared.txt

### PR DESCRIPTION
**Pull request by Nightblade**
Happy to have this PR rejected if these are considered as strictly hyphenated words.

# Add/Fix Dictionary

- Dictionary: `en-shared.txt`

## Description

- add `pregenerate` and `pregenerated`

## References

- https://en.wiktionary.org/wiki/pregenerated

## Checklist

- [X] By submitting this issue, you agree to follow our
      [Code of Conduct](https://github.com/streetsidesoftware/cspell-dicts/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: dictionaries/en_shared/dict/en-shared.txt [hunk 1]
```diff
@@ -31,6 +31,8 @@ hydrophobe
 hyponym
 hyponyms
 iPads
+pregenerate
```
Comment (Jason3S) [inline dictionaries/en_shared/dict/en-shared.txt:34]: Please add these words to [src/en-shared.txt](https://github.com/streetsidesoftware/cspell-dicts/tree/main/dictionaries/en_shared/src/en-shared.txt). They will get overwritten here.

This directory is auto generated. I should just remove it.

Comment (Jason3S) [reply dictionaries/en_shared/dict/en-shared.txt:34]: My mistake, the right location is https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/en_shared/src/shared-additional-words.txt

**Comment by Jason3S** [comment]
It might be better to add it to software terms, since that is the most likely place it is used.

**Comment by Nightblade** [comment]
Sorry about using the wrong file, I **thought** I was using the same one as my other PR.
Brain not working too well atm!

**Pull request merged**